docs/media.html

```diff
@@ -93,10 +88,6 @@ <h4 class="">Media Three</h4>
       integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
```

Comment:
Looks like you left the jQuery code, you can safely remove that in all files.

Reply:
That's a great catch. I went ahead and fixed it.